src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -17,6 +18,7 @@ client:
       pipelined: true
       weight: 1000000
       qos: 10{{end}}
+      trusted: false
```

Comment:
Why are chained servers untrusted?  At the moment all of them are run by us.  When we introduce chained peers (and if we add them to the cloud.yaml as opposed to them being discovered only by kaleidoscope) we will need to make sure to mark those as untrusted.